tests/nightly/test_large_array.py

```diff
@@ -782,8 +801,30 @@ def test_activation():
 # in future, we could test if mean, var of output
 # matches target output's mean, var
 def test_batchnorm():
-    shape = (LARGE_X, SMALL_Y)
+    def get_ref_mean_var(data, running_mean, running_var, eps, use_global_status=True):
```

Comment:
Why use ndarray to get_ref_mean_var? Wouldn't it be simpler and more correct to just implement it using numpy?

Reply:
+1 for numpy instead of nd and function could be get_np_mean_var rather than ref (i'm guessing ref mean't reference but not quite clear)

Reply:
Done